docs/multicluster/upgrade.md

```diff
@@ -0,0 +1,95 @@
+# Antrea Multi-cluster Upgrade Guide
+
+The Antrea Multi-cluster feature is introduced from v1.5.0. There is no data-plane
+related changes from release v1.5.0, so Antrea deployment and Antrea Multi-cluster
+deployment are indenpendent. However, we suggest to keep Antrea and Antrea Multi-cluster
+in the same version considering there will be data-plane change involved in the future.
+Please refer to the section `Antrea upgrade and supported version skew` in
+[Antrea Versioning](../versioning.md) to learn the requirement of Antrea upgrade.
+This doc focuses on Multi-cluster deployment only.
+
+The goal is to support 'graceful' upgrade. Multi-cluster upgrade will not have disruption
+to data-plane of member clusters, but there can be downtime of processing new configurations
+when individual components restart:
+
+- During Leader Controller restart, new member cluster, ClusterSet, ClusterClaim or
+  ResourceExport will not be processed. This is because the Controller also runs the validation
+  webhooks for MemberClusterAnnounce, ClusterSet, ClusterClaim and ResourceExport.
+- During Member Controller restart, new ClusterSet or ClusterClaim will not be processed.
+  This is because the Controller runs the validation webhooks for ClusterSet and ClusterClaim.
+
+Our goal is to support version skew for different Antrea Multi-cluster components, but
+Multi-cluster feature is still in Alpha version, the API is not stable yet, our recommendation
```

Comment:
and the API is not stable yet. Our recommendation...

Reply:
the Multi-cluster feature

Reply:
done